### DISCUSSION TITLE
Voltage on RP2040 LED-Header

### DISCUSSION THREAD

**Discussion by SezaroSteve** [ZuluSCSI/ZuluSCSI-firmware#190] [General]
Quick question: how many/much Volt is on the regular RP2040 board's LED-Header?

(It is very difficult to measure with a MultiMeter during prolonged SCSI I/O because the LED is blinking (even when it seems to light up continuously while transferring a very large file, it is not really continuously as the LED is actually switched on and off, rrrreally fast).

**Comment by aperezbios** [comment]
@SezaroSteve It's 3V. All I/O within the board happens at a nominal ~3.0V

**Reply by SezaroSteve** [reply]
ok tnx.
